test/plugin/base_test.rb

```diff
@@ -1003,6 +1003,17 @@ def test_http_request_skip_setting_latency
     end
   end
 
+  def test_dataproc_log
```

Comment:
Can you please move this right after Cloud Functions​ tests (above `test_http_request_from_record`)?